connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1649,15 +1650,17 @@ class SparkConnectPlanner(val session: SparkSession) {
       // Protobuf-specific functions
       case "from_protobuf" if Seq(2, 3, 4).contains(fun.getArgumentsCount) =>
```

Comment:
@rangadi after this change, Is there still a scene `fun.getArgumentsCount == 3`?